wordclock_interfaces/web_interface.py

```diff
@@ -137,7 +149,7 @@ class Color(Resource):
     def get(self):
         default_plugin = web_interface.app.wclk.plugins[web_interface.app.wclk.default_plugin]
         channel_wise = lambda x: {'red': x.r, 'green': x.g, 'blue': x.b}
-        channel_wise = lambda x: {'blue': x & 255, 'green': (x >> 8) & 255, 'red': (x >> 16) & 255}
+        #channel_wise = lambda x: {'blue': x & 255, 'green': (x >> 8) & 255, 'red': (x >> 16) & 255}

```

Comment:
Changed by mistake?

Reply:
I had to comment it out otherwise I get following error:
```
  File "/home/pi/rpi_wordclock/wordclock_interfaces/web_interface.py", line 152, in <lambda>
    channel_wise = lambda x: {'blue': x & 255, 'green': (x >> 8) & 255, 'red': (x >> 16) & 255}
TypeError: unsupported operand type(s) for &: 'Color' and 'int'
```